src/coreComponents/finiteElement/elementFormulations/FiniteElementBase.hpp

```diff
@@ -32,7 +32,6 @@
 #include "mesh/NodeManager.hpp"
 #include "mesh/EdgeManager.hpp"
 #include "mesh/FaceManager.hpp"
-#include "mesh/CellElementSubRegion.hpp"
```

Comment:
The three includes above it should also be removed

Reply:
Aren't they required by the following abstract initialization method?

https://github.com/GEOSX/GEOSX/blob/8000c6567184c3502d22bec74069b3302a1cee29/src/coreComponents/finiteElement/elementFormulations/FiniteElementBase.hpp#L116-L136

Reply:
You're right, I didn't spot it. Although we could in principle forward declare them - by the time the body of `fillMeshData` (where they are used) is instantiated, full definitions will be available in the calling scope. But it's not a great solution either. I just don't like that we still have an implicit circular dependency `finiteElement <-> mesh` now through the use of these mesh objects. But this probably requires some design rethinking.